src/main/java/Duke/Duke.java

```diff
@@ -0,0 +1,123 @@
+package Duke;
+
+import java.util.Scanner;
+
+public class Duke {
+    private static Task[] tasks = new Task[100];
+    private static int numberOfTasks;
+
+    private static String logo = "     ____        _        \n" +
+            "    |  _ \\ _   _| | _____ \n" +
+            "    | | | | | | | |/ / _ \\\n" +
+            "    | |_| | |_| |   <  __/\n" +
+            "    |____/ \\__,_|_|\\_\\___|\n";
+
+    private static String SEPARATION_LINE = "    ____________________________________________________________";
+
+    private static String SPACES = "       ";
+
+    private static String GREET_MESSAGE = logo + "\n" + SEPARATION_LINE + "\n" + SPACES + "Hello! I'm Duke\n" +
+            SPACES + "What can I do for you?\n" + SEPARATION_LINE + "\n";
+    private static String EXIT_MESSAGE = SEPARATION_LINE + "\n" + "     Bye. Hope to see you again soon!\n" +
+            SEPARATION_LINE;
+
+    private static char NOT_DONE_SYMBOL = '\u2718';
+
+
+    public static void main(String[] args) {
+        greet();
+
+        Scanner in = new Scanner(System.in);
+        String userInput = "";
+        boolean isInvalid = true;
+
+        while (isInvalid) {
+            userInput = in .nextLine();
+            if (!userInput.equals("list")) {
+                isInvalid = false;
+            } else {
+                printSeparationLine();
+                System.out.println("You do not have any task to do.");
+                printSeparationLine();
+            }
+        }
+
+        while (true) {
+            if (userInput.startsWith("todo ")) {
```

Comment:
Consider extracting out command parsing into a method on its own.

Reply:
You are right! Will do!